src/components/Shared/Tables/FilterableTableComponent.tsx

```diff
@@ -162,37 +168,42 @@ export class FilterableTableComponent extends React.Component<FilterableTableCom
                 key={columnHeader.name}
                 name={columnHeader.name}
                 columnHeaderCellRenderer={(columnIndex: number) => this.renderColumnHeaderCell(columnIndex, columnHeader)}
-                cellRenderer={columnData?.length ? (rowIndex, columnIndex) => this.renderCell(rowIndex, columnIndex, columnData) : undefined}
+                cellRenderer={columnData?.length ? (rowIndex, columnIndex) => this.renderCell(rowIndex, columnIndex, columnData, columnHeader) : undefined}
             />
         );
     };
 
-    private renderCell = (rowIndex: number, columnIndex: number, columnData: Array<any> | NodeJS.TypedArray) => {
+    private renderCell = (index: number, columnIndex: number, columnData: any, columnHeader: CARTA.CatalogHeader) => {
         const dataIndex = this.props.selectedDataIndex;
-
-        let contents: any;
-        if (rowIndex < columnData.length) {
-            contents = columnData[rowIndex];
-            if (typeof contents === "boolean") {
-                contents = contents.toString();
-            }
-        } else {
-            contents = "";
+        let rowIndex = index;
```

Comment:
@panchyo0 here's the issue with #1593 ...see the handling of boolean columns in the previos commit has been removed by this PR